meos/src/general/type_round.c

```diff
@@ -880,3 +880,4 @@ tnpoint_round(const Temporal *temp, Datum size)
 }
 
 /*****************************************************************************/
+#endif
```

Comment:
Github is showing a weird stop sign and colouring this line red.
Not sure why this is the case, maybe a special invisible character or line ending?
Maybe EOF character is missing?

Reply:
I've added a new line and I don't see the stop sign anymore. So it was probably that